content/adventures/en.yaml

```diff
@@ -2435,11 +2442,6 @@ adventures:
                     {if} name != 'Hedy'
                         {print} 'You are not Hedy'
                     ```
-
-                    {if} age < 13
```

Comment:
Yeah this part was just there in error I think!

Reply:
Great, thank you! I'll get back to work!